src/DiscordBot.py

```diff
@@ -35,83 +45,84 @@ async def scheduled_task(
     github_utilities: GitHubUtilities, internship_utilities: InternshipUtilities
 ):
     """
-    A scheduled task that runs every 60 seconds to check for new commits in the GitHub repository
-
-    Parameters:
-        - github_utilities: An instance of the GitHubUtilities class
-        - internship_utilities: An instance of the InternshipUtilities class
+    A scheduled task that runs every 60 seconds to check for new commits in the GitHub repository.
     """
+    start_time = datetime.now()
     try:
-        start_time = current_date = datetime.now()
         channel = bot.get_channel(int(CHANNEL_ID))
         repo = github_utilities.createGitHubConnection()
         last_saved_commit = github_utilities.getSavedSha(repo)
 
         if github_utilities.isNewCommit(repo, last_saved_commit):
-            print("New commit has been found. Finding new jobs...")
+            logging.info("New commit has been found. Finding new jobs...")
             github_utilities.setComparison(repo)  # Get the comparison file
 
             if internship_utilities.is_coop:
                 job_postings = github_utilities.getCommitChanges("README-Off-Season.md")
                 await internship_utilities.getInternships(
-                    channel, job_postings, current_date, False
+                    channel, job_postings, start_time, False
                 )
             if internship_utilities.is_summer:
                 job_postings = github_utilities.getCommitChanges("README.md")
                 await internship_utilities.getInternships(
-                    channel, job_postings, current_date, True
+                    channel, job_postings, start_time, True
                 )
             github_utilities.setNewCommit(github_utilities.getLastCommit(repo))
-            print("There were", internship_utilities.total_jobs, " new jobs found!")
+            logging.info(
+                f"There were {internship_utilities.total_jobs} new jobs found!"
+            )
 
             # Clear all the cached data
             internship_utilities.clearJobLinks()
             internship_utilities.clearJobCounter()
             github_utilities.clearComparison()
 
-            print("All jobs have been posted!")
+            logging.info("All jobs have been posted!")
         else:
-            print("No new jobs! Time: ", datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
-
-    except Exception as e:
-        traceback.print_exc()
+            logging.info(
+                f"No new jobs! Time: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}"
```

Comment:
Please include the current_date here. We are more interested in what the time was when the task started

Reply:
Will use `start_time`